source/translator.rst

```diff
@@ -0,0 +1,162 @@
+Translator Documentation
+===============================
+
+=================
+Description
+=================
+
+Translator can be considered a intermediary between game and reinforcement learning models. You can also say
```

Comment:
Zmieniłbym desc na bardziiej IS a nie CAN BE, cos w stylu, że tlumacz służy do translacji wyjścia gry na wejście do modelu i potrzebujemy go....,

Reply:
Taki styl bardziej oznajmiający, niż tłumaczący

Reply:
Fix